src/data/javascript-quiz.ts

```diff
@@ -1968,6 +1968,18 @@ const javascriptQuiz = [
     Explanation:
       "An IIFE or Immediately Invoked Function Expression, is used to declare a private variable and to avoid polluting the global environment. If a function is declared with an IIFE by using a parenthesis around it, the variable declared inside it becomes private and its value can be accessed using a getter function inside the original function.",
     Link: "https://developer.mozilla.org/en-US/docs/Glossary/IIFE"
+  },
+  {
+    Question:
+      "In JavaScript, which of the following statements is correct regarding async await?",
+    Answer: "async automatically returns a promise",
+    Distractor1: "await keeps executing code regardless of promies status",
+    Distractor2: "async can be used independently without an async function",
+    Distractor3:
+      "Multiple await keywords can be used in a single line inside an async function",
+    Explanation:
+      "An async function declaration creates an AsyncFunction object. Each time when an async function is called, it returns a new Promise which will be resolved with the value returned by the async function, or rejected with an exception uncaught within the async function.",
```

Comment:
we want to avoid copying directly from 3rd party resources.
It is always best to reword it in your own words

```suggestion
      "Declaring an asynchronous function generates an AsyncFunction object. Whenever this async function is invoked, it produces a fresh Promise. This Promise will either resolve with the value returned by the async function or reject if there's an uncaught exception within the async function.",
```

Reply:
gotcha, I will note it going forward, thank you for correcting it!